draft-ietf-acme-acme.md

```diff
@@ -812,9 +812,9 @@ identifier (required, dictionary of string):
   : The identifier itself.
 
 status (required, string):
-: The status of this authorization.  Possible values are: "unknown", "pending",
-"processing", "valid", "invalid" and "revoked".  If this field is missing, then
-the default value is "pending".
+: The status of this authorization.  Possible values are: "pending", "processing",
```

Comment:
On 09/08/2016 10:12 AM, bifurcation wrote:
> In draft-ietf-acme-acme.md
> <https://github.com/ietf-wg-acme/acme/pull/186#discussion_r78048809>:
> 
>> @@ -812,9 +812,9 @@ identifier (required, dictionary of string):
>>    : The identifier itself.
>>  
>>  status (required, string):
>> -: The status of this authorization.  Possible values are: "unknown", "pending",
>> -"processing", "valid", "invalid" and "revoked".  If this field is missing, then
>> -the default value is "pending".
>> +: The status of this authorization.  Possible values are: "pending", "processing",
> 
> I think we can probably remove "processing" here too. I originally had
> this idea that we would want to express whether the thing was waiting on
> client action or server action, but I'm not seeing any cases here where
> there would be ambiguity.
> 

I think if a server were to implement validation retries for some period
it might make sense to have a 'processing' status that would be used
during that time. Not sure if that is really that useful though...

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ietf-wg-acme/acme/pull/186/files/e89901f81b6a4f1e1f9e83a530eea6cdf76da948#r78048809>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE2GYp-CJ4Xt5gebGInyRop3GvhbUWf6ks5qoEH3gaJpZM4J2JR1>.
>